packages/shared/tailwind/overlay.ts

```diff
@@ -0,0 +1,142 @@
+const overlayColors = {
```

Comment:
Since this is a temporary file, decided to make it raw to avoid confusion down the line.

Reply:
Maybe add a todo here to remove this once the colors are fixed with the new ones.